# Translations template for ironic.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the ironic project.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: Ironic\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-08-08 06:10+0000\n"
"PO-Revision-Date: 2014-07-21 23:13+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: German (http://www.transifex.com/projects/p/ironic/language/"
"de/)\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ironic/drivers/modules/agent.py:421
#, python-format
msgid "Async exception for %(node)s: %(msg)s"
msgstr ""

#: ironic/drivers/modules/ipminative.py:345
#, python-format
msgid ""
"IPMI set boot device failed for node %(node_id)s with the following error: "
"%(error)s"
msgstr ""

#: ironic/drivers/modules/ipminative.py:380
#, python-format
msgid ""
"IPMI get boot device failed for node %(node_id)s with the following error: "
"%(error)s"
msgstr ""

#: ironic/drivers/modules/seamicro.py:538
#, python-format
msgid ""
"Seamicro set boot device failed for node %(node)s with the following error: "
"%(error)s"
msgstr ""

#: ironic/drivers/modules/ssh.py:598
#, python-format
msgid ""
"Failed to set boot device for node %(node)s, virt_type %(vtype)s does not "
"support this operation"
msgstr ""

#: ironic/nova/virt/ironic/driver.py:296
#, python-format
msgid ""
"Failed to clean up the parameters on node %(node)s when unprovisioning the "
"instance %(instance)s"
msgstr ""

#: ironic/nova/virt/ironic/driver.py:539
#, python-format
msgid ""
"Error preparing deploy for instance %(instance)s on baremetal node %(node)s."
msgstr ""

#: ironic/nova/virt/ironic/driver.py:564
#, python-format
msgid "Error deploying instance %(instance)s on baremetal node %(node)s."
msgstr ""

#: ironic/openstack/common/excutils.py:76
#, python-format
msgid "Original exception being dropped: %s"
msgstr "Ursprüngliche Ausnahme wird gelöscht: %s"

#: ironic/openstack/common/excutils.py:105
#, python-format
msgid "Unexpected exception occurred %d time(s)... retrying."
msgstr "Unerwartete Ausnahme %d mal(e) aufgetreten... Neuversuch."

#: ironic/openstack/common/lockutils.py:119
#, python-format
msgid "Could not release the acquired lock `%s`"
msgstr ""

#: ironic/openstack/common/loopingcall.py:95
msgid "in fixed duration looping call"
msgstr "in Schleifenaufruf mit festgelegter Dauer"

#: ironic/openstack/common/loopingcall.py:138
msgid "in dynamic looping call"
msgstr "in dynamischen Schleifenaufruf"

#: ironic/openstack/common/periodic_task.py:202
#, python-format
msgid "Error during %(full_task_name)s: %(e)s"
msgstr ""

#: ironic/openstack/common/service.py:188
msgid "Exception during rpc cleanup."
msgstr ""

#: ironic/openstack/common/service.py:277
msgid "Unhandled exception"
msgstr ""
